.gitmodules

```diff
@@ -0,0 +1,19 @@
+[submodule "local_deps/h2"]
```

Comment:
I don't think we need the submodule to fix this issue.

Reply:
This submodule does not exist anymore on last commit, that is what resolves the problem

Reply:
It still remains. That's why I can review this file. Please squash your commits.